src/app/find/mapPage/MapPage.jsx

```diff
@@ -23,7 +24,7 @@ const initialFiltersState = {
 export default class MapPage extends Component {
   state = {
     initialLocationsLoaded: false,
-    locations: null,
+    locations: [],
```

Comment:
Starting with an empty array means we don't necessarily have a way to distinguish the case where no search has happened yet from the case a search has happened, but returned nothing. That could e.g. lead to a brief flash of the "no results" bar when the page loads, if it renders before the first search is dispatched.

I don't think it matters much here, but generally it can help to distinguish _not having data_ from having data and _knowing it's empty_, and null can be useful for the former.